Updated Netdata documentation

noodlemctwoodle opened the pull request:
**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Commit:
Updated Netdata documentation

noodlemctwoodle commented:
New pull request for issue posted here 
Updated Example configuration for Netdata #9463

Commit:
Added missing `sensor:` tag

Commit:
Added further examples and rephrased wording

Commit:
Fixed `JSON` display issue

fabaff commented on source/_components/netdata.markdown:
```diff
@@ -43,23 +43,71 @@ $ curl -X GET "http://[Netdata_Instance]:19999/api/v1/data?chart=[data_group]&po
 ## {% linkable_title Configuration %}
 
-To add this platform to your installation, add the following to your `configuration.yaml` file:
+To add this component to your installation, add the following to your `configuration.yaml`.
```
Per definition is this integration a sensor platform. Also, it's a configuration file.

noodlemctwoodle replied:
Changed wording for adding to the configuration.yaml

fabaff commented on source/_components/netdata.markdown:
```diff
@@ -43,23 +43,71 @@ $ curl -X GET "http://[Netdata_Instance]:19999/api/v1/data?chart=[data_group]&po
-    resources:
-      load:
+    resources:    
+      system_load:
```
There is no need to change that. It's the name and it can be different.

noodlemctwoodle replied:
reverted `system_load` to `load`

fabaff commented on source/_components/netdata.markdown:
````diff
@@ -43,23 +43,71 @@ $ curl -X GET "http://[Netdata_Instance]:19999/api/v1/data?chart=[data_group]&po
+        data_group: 'cpu.cpufreq'
+        element: 'cpu0'
+        icon: mdi:chip
+```
````
Full examples should be placed after the basic configuration sample -> https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

Remove the comments please. They are superfluous because the keys are described below.

noodlemctwoodle replied:
Removed comments in the example and added example below the basic configuration sample.

Commit:
Fixed issues raised by fabaff

reverted `system_load` to `load`
modified full example
removed comments from example

klaasnicolaas committed:
:pencil2: Tweak

noodlemctwoodle commented:
Are any further changes required?

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas committed:
:pencil2: Tweak
